src/components/common/error-message/error-message.module.ts

```diff
@@ -1,17 +1,17 @@
-import { NgModule } from '@angular/core';
-import { IonicModule } from '@ionic/angular';
-import { ErrorMessageComponent } from './error-message';
-
-@NgModule({
-  declarations: [
-    ErrorMessageComponent,
-  ],
-  imports: [
-    IonicModule,
-  ],
-  providers: [],
-  exports: [
-    ErrorMessageComponent,
-  ],
-})
-export class ErrorMessageModule { }
+// import { NgModule } from '@angular/core';
+// import { IonicModule } from '@ionic/angular';
+// import { ErrorMessageComponent } from './error-message';
+//
+// @NgModule({
+//   declarations: [
+//     ErrorMessageComponent,
+//   ],
+//   imports: [
+//     IonicModule,
+//   ],
+//   providers: [],
+//   exports: [
+//     ErrorMessageComponent,
+//   ],
+// })
+// export class ErrorMessageModule { }
```

Comment:
this can be deleted now

Reply:
i commented it out to confirm it wasnt needed. good spot, removed now.